app/src/controllers/profileController.js

```diff
@@ -55,7 +55,18 @@ export default class ProfileController extends ControllerBase {
               // Look if the response is a GetProfileOKResponse.
               validateResponse(maybeApiProfile, GetProfileOKResponseModel).fold(
                 // Look if object is a ProblemJson.
-                () => validateProblemJson(maybeApiProfile, res),
```

Comment:
there's [many ways](https://github.com/gcanti/fp-ts/blob/master/src/Either.ts) you can compose `Either`, I wonder if there's a more elegant way that nesting folds

Reply:
I'm so expert in FP to provide a good answer to this question :-) . I can try to find a more elegant way to do this.

Reply:
no worries, it's not a big deal :)